### DISCUSSION TITLE
Can `props.children` only be used in the template?

### DISCUSSION THREAD

**Discussion by brandonpittman** [BuilderIO/jsx-lite#1300] [Q&A] [answered]
I can display `props.children` in a component's template, but it's not available generally off of the `props` value. I want to check the string in the children for other attributes.

**Comment by samijaber** [comment, marked as answer]
The concept of `children` is not the same across all frameworks. Some of them (like Qwik) can't access them outside the render at all. So yeah, it can only be safely used in the template (only to be rendered).
